client/src/Bible.api.ts

```diff
@@ -1,177 +1,160 @@
+import {
```

Comment:
i wonder why this got reformatted? did you edit it manually? (it should not be edited)